client/src/components/Tool/ToolForm.vue

```diff
@@ -224,6 +224,7 @@ export default {
         },
         onValidation(validationInternal) {
             this.validationInternal = validationInternal;
+            this.validationInternal = validationInternal;
```

Comment:
```suggestion
```

Reply:
that's probably a merge artifact, thanks!